osclib/stagingapi.py

```diff
@@ -1757,6 +1757,11 @@ def create_adi_project(self, name, use_frozenlinks=False, src_prj=None):
             linkproject = ''
             repository = '<repository name="standard">'
 
+        if nonfree_repo_required:
+            nonfree_repo = "<path project=\"{}\" repository=\"standard\"/>".format(self.cnonfree)
```

Comment:
nit: seems like `nonfree_path` would make more sense given it is a path inside repo not a new repo.